.github/scripts/notifyAboutIconRequest.js

```diff
@@ -0,0 +1,32 @@
+// @ts-nocheck
```

Comment:
Why is this set to nocheck?

Reply:
Copied this over from another sample, removed it from this PR.